app/bundles/CampaignBundle/Controller/CampaignController.php

```diff
@@ -673,14 +687,31 @@ protected function getViewArguments(array $args, $action)
                 $dateRangeValues = $this->request->get('daterange', []);
                 $action          = $this->generateUrl('mautic_campaign_action', ['objectAction' => 'view', 'objectId' => $objectId]);
                 $dateRangeForm   = $this->get('form.factory')->create(DateRangeType::class, $dateRangeValues, ['action' => $action]);
+                $events          = $this->getCampaignModel()->getEventRepository()->getCampaignEvents($entity->getId());
+                $dateFrom        = null;
+                $dateTo          = null;
+                $dateToPlusOne   = null;
+                $this->setCoreParametersHelper($this->get('mautic.config'));
+                if ($this->coreParametersHelper->get('campaign_by_range')) {
+                    $dateFrom        = new \DateTimeImmutable($dateRangeForm->get('date_from')->getData());
+                    $dateTo          = new \DateTimeImmutable($dateRangeForm->get('date_to')->getData());
+                    $dateToPlusOne   = $dateTo->modify('+1 day');
+                }
 
                 /** @var LeadEventLogRepository $eventLogRepo */
-                $eventLogRepo             = $this->getDoctrine()->getManager()->getRepository('MauticCampaignBundle:LeadEventLog');
-                $events                   = $this->getCampaignModel()->getEventRepository()->getCampaignEvents($entity->getId());
-                $leadCount                = $this->getCampaignModel()->getRepository()->getCampaignLeadCount($entity->getId());
-                $campaignLogCounts        = $eventLogRepo->getCampaignLogCounts($entity->getId(), false, false, true);
+                $eventLogRepo             = $this->getDoctrine()->getManager()->getRepository(LeadEventLog::class);
                 $pendingCampaignLogCounts = $eventLogRepo->getCampaignLogCounts($entity->getId(), false, false);
```

Comment:
I'm a bit afraid that this is loading the count form the statistical table instead of the summary table no matter if the `campaign_use_summary` config option is true or false. Can you think of a way how to get this pending count to the summary table as well?

Reply:
I think I have a better solution to this, the pending counts are calculated based on total leads - log entries. We can use similar logic without use of `pendingCount` column in summary table.

Reply:
It returns the same results on my test campaign. Good thinking!